builtin_interfaces/Time stamp
int64 ndx
int64 nu
std_msgs/Float64MultiArray k0
std_msgs/Float64MultiArray xs
std_msgs/Float64MultiArray us
bool[] contact_states
float64[] forces
float64[] a0
float64 process_duration